.changeset/hot-actors-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/api": patch
+---
+
+Drop nodemailer-sendgrid support since the package is no longer maintained
```

Comment:
This is a major change to the `@directus/api` package

```suggestion
---
"@directus/api": major
---

Dropped support for the SendGrid email transport option
```

Reply:
As suggested I made the changes